### PR TITLE
Updated frame.replace()

### PR DESCRIPTION
**Pull request by hsubbaraj**
Fix for replace to add support for the mapping case as referenced in #495 :

`kdf.replace({0: 10, 1: 100})`

### CODE REVIEW

**Comment by hsubbaraj** [comment]
@ueshin can you take a look? Thanks!

File: databricks/koalas/tests/test_dataframe.py [hunk 1]
```diff
@@ -782,6 +782,27 @@ def test_replace(self):
...
         )
 
+        pdf = pd.DataFrame({'A': [0, 1, 2, 3, 4],
+                            'B': [5, 6, 7, 8, 9],
+                            'C': ['a', 'b', 'c', 'd', 'e']})
+
+        kdf = ks.from_pandas(pdf)
+
+        assert(repr(kdf.replace([0, 1, 2, 3], 4)) ==
+               repr(pdf.replace([0, 1, 2, 3], 4)))
```
Comment (ueshin) [inline databricks/koalas/tests/test_dataframe.py]: Shall we use `self.assert_eq()`?

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -25,6 +25,7 @@
 
 import numpy as np
 import pandas as pd
+import pyspark.sql.functions as F
```
Comment (ueshin) [inline databricks/koalas/frame.py]: nit: an unnecessary change?

Comment (hsubbaraj) [reply databricks/koalas/frame.py]: Yes, I didn't notice the import on a different line!

File: databricks/koalas/tests/test_dataframe.py (hunk 1 above)
Comment (hsubbaraj) [reply databricks/koalas/tests/test_dataframe.py]: Sure, I will make the change!

**Comment by hsubbaraj** [comment]
@ueshin should be good to go, can you take a look? Thanks :)

**Comment by ueshin** [comment]
Thanks! merging.
